ArcGISRuntimeSDKQt_QMLSamples/Analysis/ConfigureSubnetworkTrace/ConfigureSubnetworkTrace.qml

```diff
@@ -0,0 +1,423 @@
+            if (networkAttribute.domain && (networkAttribute.domain.domainType === Enums.DomainTypeCodedValueDomain)) {
+                const dataType = networkAttribute.dataType;
+                const domain = networkAttribute.domain;
+                const codedValueName = domain.codedValues[expression.value].name;
+                return "`%1` %2 `%3`".arg(networkAttribute.name).arg(operatorAsString).arg(codedValueName);
+            } else {
+                return "`%1` %2 `%3`".arg(networkAttribute.name).arg(operatorAsString).arg(otherNetworkAttribute? otherNetworkAttribute.name : expression.value)
+            }
+        case Enums.UtilityTraceConditionTypUtilityCategoryComparisone:
+            return "`%1` %2".arg(networkAttribute.category.name).arg(networkAttribute.comparisonOperator === Enums.UtilityCategoryComparisonOperatorExists ? "Exists" : "DoesNotExist");
+        case Enums.UtilityTraceConditionTypeUtilityTraceAndCondition:
+            return "`%1` AND\n `%3`".arg(expressionToString(expression.leftExpression)).arg(expressionToString(expression.rightExpression));
+        case Enums.UtilityTraceConditionTypeUtilityTraceOrCondition:
+            return "`%1` OR\n `%3`".arg(expressionToString(expression.leftExpression)).arg(expressionToString(expression.rightExpression));
+        }
+    }
+
+    function comparisonOperatorToString(comparisonOperator) {
+        switch (comparisonOperator) {
+        case Enums.UtilityAttributeComparisonOperatorEqual:
+            return "Equal";
+        case Enums.UtilityAttributeComparisonOperatorNotEqual:
+            return "NotEqual";
+        case Enums.UtilityAttributeComparisonOperatorGreaterThan:
+            return "GreaterThan";
+        case Enums.UtilityAttributeComparisonOperatorGreaterThanEqual:
+            return "GreaterThanEqual";
+        case Enums.UtilityAttributeComparisonOperatorLessThan:
+            return "LessThan";
+        case Enums.UtilityAttributeComparisonOperatorLessThanEqual:
+            return "LessThanEqual";
+        case Enums.UtilityAttributeComparisonOperatorIncludesTheValues:
+            return "IncludesTheValues";
+        case Enums.UtilityAttributeComparisonOperatorDoesNotIncludeTheValues:
+            return "DoesNotIncludeTheValues";
+        case Enums.UtilityAttributeComparisonOperatorIncludesAny:
+            return "IncludesAny";
+        case Enums.UtilityAttributeComparisonOperatorDoesNotIncludeAny:
+            return "DoesNotIncludeAny";
+        default:
+            return null;
+        }
+    }
+
+    function addCondition() {
+        // NOTE: You may also create a UtilityCategoryComparison with UtilityNetworkDefinition.Categories and UtilityCategoryComparisonOperator.
+
+        if (traceConfiguration === null) {
+            traceConfiguration = ArcGISRuntimeEnvironment.createObject("UtilityTraceConfiguration");
+        }
+        if (traceConfiguration.traversability === null) {
```

Comment:
Also, since this is a standalone clause, I would put a line between this if-statement and the previous one. When they're grouped like this my mind assumes the second one is an else-if and they are connected.